`next/compat/router` (#42502)
vercel/next.js: errors/next-router-not-mounted.md

# NextRouter was not mounted

#### Why This Error Occurred

> A component used `useRouter` outside a Next.js application, or was rendered outside a Next.js application. This can happen when doing unit testing on components that use the `useRouter` hook as they are not plumbed into Next.js.

Reviewer: ```suggestion
A component used `useRouter` outside a Next.js application, or was rendered outside a Next.js application. This can happen when doing unit testing on components that use the `useRouter` hook as they are not configured with Next.js' contexts.
```
